### PR TITLE
Responsive design fixes

### PR DESCRIPTION
**Pull request by kolaente**
This fixes some issues which annoyed me the most in the responsive version of Gitea.

There's probably more to do, but these fixes are the ones I found the most obvious.

### Screenshots

screen shot 2018-07-24 at 11 55 50
screen shot 2018-07-24 at 11 56 00
screen shot 2018-07-24 at 11 56 18
screen shot 2018-07-24 at 11 56 30
screen shot 2018-07-24 at 11 58 45-fullpage
screen shot 2018-07-24 at 11 59 49-fullpage
screen shot 2018-07-24 at 12 00 03-fullpage
screen shot 2018-07-24 at 12 00 20-fullpage
screen shot 2018-07-24 at 12 00 32-fullpage

### CODE REVIEW

**Comment by kolaente** [comment]
There seems to be an issue with the commits, I pushed one commit, not 382... Should I resolve this somehow?

**Comment by jonasfranz** [comment]
Please `reset --hard upstream/master` and cherry pick your commits. Please make sure to `reset --hard` on a copy of your branch since it will delete all your commits. Then force push it to your remote branch.

**Comment by kolaente** [comment]
@JonasFranzDEV Done.

File: public/less/_dashboard.less [hunk 1]
```diff
@@ -26,6 +26,13 @@
                     top: 7px;
                     left: 90%;
                     width: 15%;
+
+					@media only screen and (max-width: 768px) {
```
Comment (lafriks) [inline public/less/_dashboard.less]: spacing seems to be off

**Review by lafriks — CHANGES_REQUESTED** [review]
Spacing is wrong in added lines

File: public/less/_dashboard.less (hunk 1 above)
Comment (kolaente) [reply public/less/_dashboard.less]: Fixed, apparantly my editor was set to use tabs instead of spaces as the rest of the file.

**Review by lafriks — APPROVED** [review]

**Review by Wqrld — APPROVED** [review]

**Review by jonasfranz — APPROVED** [review]

**Pull request merged**